docs/registry/attributes/otel.md

```diff
@@ -101,6 +102,18 @@ E.g. for Java the fully qualified classname SHOULD be used in this case.
 | `simple_span_processor` | The builtin SDK simple span processor | ![Development](https://img.shields.io/badge/-development-blue) |
 | `zipkin_http_span_exporter` | Zipkin span exporter over HTTP | ![Development](https://img.shields.io/badge/-development-blue) |
 
+## OTel Event Attributes
+
+Attributes used by non-OTLP exporters to represent OpenTelemetry Event's concepts.
+
+**Attributes:**
+
+| Key | Stability | Value Type | Description | Example Values |
+|---|---|---|---|---|
+| <a id="otel-event-name" href="#otel-event-name">`otel.event.name`</a> | ![Development](https://img.shields.io/badge/-development-blue) | string | Identifies the class / type of event. [3] | `browser.mouse.click`; `device.app.lifecycle` |
+
+**[3] `otel.event.name`:** This attribute SHOULD be used by non-OTLP exporters when destination does not support `EventName` or equivalent field. This attribute MAY be used by applications using existing logging libraries so that it can be used to set the `EventName` field by Collector or SDK components.
```

Comment:
Question on the second part: If a user does logging with a 3rd party logging library, and sets an attribute with `otel.event.name`, will that get stored in OTel Sdk LogRecord as the top-level "EventName" automatically? Or it requires additional LogRecordProcessor/similar to do that promotion?

Reply:
> will that get stored in OTel Sdk LogRecord as the top-level "EventName" automatically? Or it requires additional LogRecordProcessor/similar to do that promotion?

I would leave it to the language to decide what is better for them. Personally, I think that doing it via `LogRecordProcessor` is "better code reuse" (no need to apply the same logic in each logging bridge) and also gives more flexibility (you can decide for which loggers/libraries you apply this logic).